# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/semantics/scripting-1/the-script-element/moving-between-documents/before-prepare-iframe-parse-error-external-classic.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/html/semantics/scripting-1/the-script-element/moving-between-documents/before-prepare-iframe-parse-error-external-classic.html.ini
--- a/testing/web-platform/meta/html/semantics/scripting-1/the-script-element/moving-between-documents/before-prepare-iframe-parse-error-external-classic.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/html/semantics/scripting-1/the-script-element/moving-between-documents/before-prepare-iframe-parse-error-external-classic.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,7 +1,8 @@
 [before-prepare-iframe-parse-error-external-classic.html]
   [<script> error: Move parse-error external classic script to iframe before-prepare]
     expected:
+      if (os == "linux") and not webrender and not debug and (processor == "x86"): ["PASS", "FAIL"]
+      if (os == "linux") and not webrender and not debug and (processor == "x86_64"): ["PASS", "FAIL"]
       if (os == "linux") and not webrender and debug: ["PASS", "FAIL"]
-      if (os == "linux") and not webrender and not debug: ["PASS", "FAIL"]
       if (os == "linux") and webrender: ["PASS", "FAIL"]
 